staff/templates/staff/base.html

```diff
@@ -0,0 +1,57 @@
+{% load django_vite %}
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en" class="no-js min-h-screen {% block html_class %}{% endblock html_class %}">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>{% block metatitle %}{% endblock metatitle %}</title>
+  <meta name="description" content="{% block meta_description %}{% endblock meta_description %}">
+
+  <link rel="manifest" href="{% static "manifest.webmanifest" %}">
+  <link rel="icon" href="{% static "favicon.ico" %}" sizes="any">
+  <link rel="icon" href="{% static "icon.svg" %}" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="{% static "apple-touch-icon.png" %}">
+
+  {% vite_hmr_client %}
+  {% vite_asset 'assets/src/scripts/bootstrap.js' defer=true %}
+</head>
+<body class="d-flex flex-column min-vh-100">
+  <nav class="navbar navbar-expand-lg navbar-dark bg-danger py-lg-2">
```

Comment:
Controversial opinion: I don't think we need the "staff area" banner at the very top of the page.

There's a large pink jumbotron on each of the pages to indicate that we're in the danger zone and the navigation within that is so easy/convenient that I don't think the links at the very top are needed.

Reply:
I can definitely see that while we've only got 3, very related, objects.  I use them quite a bit in job-server.  Can we keep them so that the UI matches job-server and then when Tom gets round to redesigning the staff area we can use that design in both places?  I'd love to merge them into the existing banner or do _something_ so we don't have a whole extra menu bar!